app/forms.py

```diff
@@ -1,27 +1,13 @@
-from django import forms
-from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.forms import UserChangeForm, UserCreationForm
 
 from .models import User
 
-class SignUpForm(UserCreationForm):
-    email = forms.EmailField(max_length=254, help_text='Required. Inform a valid email address.')
 
-    class Meta:
+class UserChangeForm(UserChangeForm):
+    class Meta(UserChangeForm.Meta):
```

Comment:
Is it needed to inherit from `UserChangeForm.Meta`?

Reply:
yes i dont need write every thing from zero just i inherited from UserChangeForm.Meta and customize User Model

Reply:
Ok, checked it and you're right.